microcosm_daemon/tests/test_daemon.py

```diff
@@ -0,0 +1,28 @@
+"""
+Test daemon loading.
+
+"""
+from hamcrest import assert_that, equal_to, is_
+
+from microcosm_daemon.daemon import Daemon
+
+
+class FixtureDaemon(Daemon):
+
+    @property
+    def name(self):
+        return "fixture"
+
+    @property
+    def components(self):
+        return super(FixtureDaemon, self).components + [
+            "hello_world",
+        ]
+
+    def __call__(self):
+        pass
+
+
+def test_daemon_initialize():
```

Comment:
One of our common failure modes for daemons is that we write tests for their _handlers_ but not for their initialization. This means that some easy-to-avoid configuration errors don't get caught until deploy.

Now we can (and should!) validate that every daemon's object graph loads at unit test time.